packages/embed-chart/public/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```

Comment:
I can't comment directly b/c they're binary files, but I think the `public/favicon.ico` and `public/logo` are the react-scripts generated files and can be deleted

Reply:
They're actually Deephaven logos, but they're not used so I'll delete them.